src/snowcli/output/types.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import annotations
+import json
+import typing as t
+
+from snowflake.connector.cursor import SnowflakeCursor
+
+
+class Object:
```

Comment:
Maybe all classes from this module should have prefix `Result`? WDYT?

I think Message and Stream are too general

Reply:
How about suffix instead of prefix? `MessageResult`, `StreamResult` or `SingleQueryResult` wdyt?

Reply:
Yes, I meant suffix 🤦